app/assets/javascripts/admin/routes/admin-web-hook.js.es6

```diff
@@ -0,0 +1,15 @@
+import WebHook from 'admin/models/web-hook';
+
+export default Discourse.Route.extend({
+  serialize(model) {
+    return { web_hook_id: model.get('id') };
+  },
+
+  model(params) {
```

Comment:
Let's use custom store instead, it'll save you from alot of the glue that you have to write.

https://meta.discourse.org/t/upgrading-our-front-end-models-to-use-a-store/27837